examples/axp_optimizer_example.py

```diff
@@ -0,0 +1,140 @@
+            id="CLIENT:/OPTIMIZER/IBM.ACCT")
+        axp_optimizer_optimization = Optimization(
+            risk_model_date="09/01/2020",
+            backtest_date="09/01/2020",
+            cashflow="0"
+        )
+        axp_optimizer_trades_list = OptimizerTradesList(
+            identifier_type="SedolChk", include_cash=False)
+        axp_optimizer_output_types = OptimizerOutputTypes(
+            trades=axp_optimizer_trades_list)
+        axp_optimizer_parameters = AxiomaEquityOptimizationParameters(
+            strategy=axp_optimizer_strategy,
+            output_types=axp_optimizer_output_types,
+            account=axp_optimizer_account,
+            optimization=axp_optimizer_optimization
+        )
+        axp_optimization_parameters_root = AxiomaEquityOptimizationParametersRoot(
+            data=axp_optimizer_parameters)
+
+        axp_optimizations_api = AXPOptimizerApi(api_client)
+
+        post_and_optimize_response = axp_optimizations_api.post_and_optimize(
+            axioma_equity_optimization_parameters_root=axp_optimization_parameters_root,
+            _return_http_data_only=False
+        )
+
+        if post_and_optimize_response[1] == 201:
+            output_optimization_result(post_and_optimize_response[0]['data'])
+        else:
+            optimization_id = post_and_optimize_response[0].data.id
+            print("Calculation Id: " + optimization_id)
+
+            status_response = axp_optimizations_api.get_optimization_status_by_id(id=optimization_id,
+                                                                                  _return_http_data_only=False)
+
+            while status_response[1] == 202:
+                max_age = '5'
+                age_value = status_response[2].get("cache-control")
+                if age_value is not None:
+                    max_age = age_value.replace("max-age=", "")
+                print('Sleeping: ' + max_age)
+                time.sleep(int(max_age))
+                status_response = axp_optimizations_api.get_optimization_status_by_id(optimization_id,
+                                                                                      _return_http_data_only=False)
+
+            if status_response[1] == 201:
+                print("Optimization Id: " + optimization_id + " Succeeded!!!")
+                result_response = axp_optimizations_api.get_optimization_result(id=optimization_id,
+                                                                                _return_http_data_only=False)
+                output_optimization_result(result_response[0]['data'])
+            else:
+                print("Optimization Id:" + optimization_id + " Failed!!!")
+                print("Error message : " + status_response[0].error)
+
+    except ApiException as e:
+        print("Api exception Encountered")
+        print(e)
+        exit()
+
+
+def output_optimization_result(result):
+    print("Optimization Result")
+    print(result)
```

Comment:
For optimizers, we need to run a for-loop on the tables list. Can you please refer to Java and implement the same here?

Reply:
I was looking at the code for https://github.com/factset/analyticsapi-engines-java-sdk/blob/master/examples/src/examples/BpmInteractiveOptimizerEngineExample.java#L145. I don't know how this (https://github.com/factset/analyticsapi-engines-python-sdk/pull/34/files/d18826bfc95b9ffff31d8f26ec04b85a29bcce0b#diff-9a660acba24f88fd0f1c5cf8ed7ec9e86b79b804bc7c3a0bd715c929bedad2dbR130) differs from that.

Reply:
done